project-base/app/translations/messages.cs.po

```diff
@@ -1295,113 +1346,116 @@ msgid "Paid amount"
 msgstr "Zaplacená částka"
 
 msgid "Parameter"
-msgstr ""
+msgstr "Parametr"
+
+msgid "Parameter value"
+msgstr "Hodnota parameteru"
+
+msgid "Parameter value <strong><a href=\"{{ url }}\">{{ parameterValue.text }}</a></strong> was successfuly edited."
+msgstr "Hodnota parametru <strong><a href=\"{{ url }}\">{{ parameterValue.text }}</a></strong> je úspěšně upravena."
+
+msgid "Parameter value of type color"
+msgstr "Hodnota parameteru typu barva"
+
+msgid "Parameter values of type color"
+msgstr "Hodnoty parameterů typu barva"
 
 msgid "Parameters - full"
 msgstr "Parametry - vše"
 
 msgid "Parameters - view"
 msgstr "Parametry - zobrazení"
 
-msgid "Parametry a pořadí v kategorii"
-msgstr ""
-
-msgid "Parametry filtru"
-msgstr ""
+msgid "Parameters order in category"
+msgstr "Pořadí parametrů v kategorii"
 
-msgid "Parametry produktů vybrané kategorie"
-msgstr ""
+msgid "Pararameter value of type color"
+msgstr "Hodnota parametru typu barva"
 
 msgid "Password"
 msgstr "Heslo"
 
 msgid "Password again"
 msgstr "Heslo znovu"
 
+msgid "Password has to include uppercase letters, lowercase letters, numbers and must be longer than 10 characters."
+msgstr "Heslo musí obsahovat velké, malé písmena, číslice a musí být delší než 10 znaků."
+
+msgid "Payment authorized"
+msgstr "Platba předautorizována"
```

Comment:
`předautorizována` is correct? From the English translation I would say `autorizována/schválena`

Reply:
I am not sure about the implementation I used this as it was because I did not know reason behind it

Reply:
ok